week-2/14-align-self/readme.md

```diff
@@ -0,0 +1,13 @@
+# Align self
+
+With `align-items` you move all the items at once (on the **non-main axis**).
+With `align-self` you move only one of the items.
+
+![Example of align-items vs. align-self](./ignore_this_folder/example1.png)
+
+You can use the same values for `align-self` as you used for `align-items`.
+
+Exercise:
```

Comment:
Again, I'd say move this above the example image, just after the `align-items`, `align-self` descriptions. Then some more text like this:

> In this exercise, you will use `align-self` and `align-items` to change the position of the items. Follow the steps below to complete this exercise.
>
> 1. .... [like the description above]